proprietary/font/README.md

```diff
@@ -1,3 +1,36 @@
-# Assets
+# Font assets voor het LUX Design System
 
-Story any font files here, such as proprietary fonts.
+Deze package bevat de lettertypes RijksoverheidSerif en RijksoverheidSans die het LUX Design System gebruikt.  
+**Let op:** Deze fonts zijn auteurrechtelijk beschermd. Lees meer op [Gebruiksvoorwaarden typografie](https://www.rijkshuisstijl.nl/publiek/modules/product/DigitalStyleGuide/default/index.aspx?ItemId=10370) op Rijkshuisstijl.nl
+
+## Installeren
+
+### NPM
+
+`npm install @lux-design-system/font --save-dev`
```

Comment:
Ik denk niet dat wij dit mogen doen. AZ is erg beschermend over het Rijkshuisstijl font. Die moet je echt via de website van Rijkshuisstijl downloaden. Dat staat beschermd achter een login voor alleen rijksmedewerkers. Als wij dit zo aanbieden omzeilen we dat en gaan we op de vingers getikt worden.

Reply:
Ik begrijp dat ik naar elke website van de Rijksoverheid kan gaan en de fonts kan downloaden. Dat gras maai ik alvast voor je voeten weg. Het lijkt er alleen op dat de beheerder dat niet begrijpt.

Reply:
Ze staan al in onze source-code hier in github en in bijv [@nl-rvo/assets](https://www.npmjs.com/package/@nl-rvo/assets?activeTab=code)

Reply:
Maar ander kopieer ik alleen de css en zetten we in de readme dat ze ze zelf maar moeten downloaden en in de/een folder zetten oid

Reply:
> Maar ander kopieer ik alleen de css en zetten we in de readme dat ze ze zelf maar moeten downloaden en in de/een folder zetten oid

We hebben offline besproken dat dit de manier is om verder te gaan.

Reply:
Fonts worden nu niet meer meegepubliceerd. Readme is aangepast.